tests/System.Text.Primitives.Tests/Formatting/TimeTests.cs

```diff
@@ -51,6 +53,7 @@ public void RandomDateTimeTests(char format)
         static void TestDateTimeFormat(DateTime dt, char format, SymbolTable symbolTable)
         {
             var expected = dt.ToString(format.ToString(), CultureInfo.InvariantCulture);
+            if (format == 'r') expected = expected.ToLowerInvariant();
```

Comment:
Hmm, I am not sure what to do here. I could use some other format char for "lowercassed R", but 'r' seems to be perfect for it. But you are right, it creates an incompatibility with what ToString does.

@tarekgh, any feedback on this? Do you think it would be reasonable to use 'r' for lowercased format? Also, why ToString('r') does not produce lowercases output?